src/components/country-blurb/country-blurb.jsx

```diff
@@ -0,0 +1,61 @@
+const classNames = require('classnames');
+const PropTypes = require('prop-types');
+const React = require('react');
+
+require('./country-blurb.scss');
+
+const CountryBlurb = props => (
+    <div className={classNames('country-blurb', props.className)}>
+        {props.className === 'regular' &&
```

Comment:
Maybe CountryBlurb should get a more generic title or since it's likely fairly specific to the annual report, we can move it into the `src/annual-report` top level directory.

Reply:
moved into annual-report directory